packages/map/src/map/hooks.jsx

```diff
@@ -58,47 +63,47 @@ export const useMapInstance = (nodeRef, options) => {
       setMap(null);
       map.remove();
     };
-  }, [noAttribution, locked, lockedMobile]);
+  }, [noAttribution, locked, lockedMobile, hasBounds]);
 
   return mapInstance;
 };
 
 export const useMapUpdate = (map, {
   bounds,
-  childrenBounds,
   animate,
   fitPadding,
-  defaultView,
-  defaultZoom,
+  maxZoom,
 }) => {
-  const boundsToFit = bounds || childrenBounds;
-
   useEffect(() => {
-    const boundingBox = getBoundingBox(boundsToFit);
+    const boundingBox = getBoundingBox(bounds);
 
-    if (map && boundingBox && !defaultView) {
+    if (map && boundingBox) {
       const options = { animate, padding: fitPadding };
-      if (defaultZoom) options.maxZoom = defaultZoom;
+      if (maxZoom) options.maxZoom = maxZoom;
```

Comment:
do we need this option? i mean it's ok to pass it along but do we need to set it in core FE for map to work?